README.md

```diff
@@ -1,4 +1,27 @@
+## Introduction
 
+**What?** The [Neurodata Without Borders: Neurophysiology][nwb-neurophysiology] (NWB:N) Hackathon is a week-long hands
+on activity in which neurophysiology researchers create solutions using the open source NWB:N software packages.
+Participants work collaboratively on solutions to develop a unified data format for cellular-based neurophysiology data, focused on the dynamics of groups of neurons measured under a large range of experimental conditions. In contrast to conferences and workshops where the primary focus is to report results, the objective of the Hackathon is to provide a venue for creators of neurophysiology open-source software to collaboratively work.
+
+[nwb-neurophysiology]: http://www.nwb.org/nwb-neurophysiology/
+
+
+**When, where, how much?** Twice a year - Early April in Seattle, and late April in Berkley. Ad-hoc meetings are added occasionally. There is no registration fee, and lunch and dinner are usually covered.
```

Comment:
Twice a year - Early April in Seattle, and late April in Berkley. Ad-hoc meetings are added occasionally.
=>
To be determined based on participation. This year there will be a development focused Hackathon in Early April in Seattle, and then a user on-boarding tutorial Hackathon in late April in Berkeley. Ad-hoc meetings are added occasionally. We plan to have at least one more Hackathon in 2019.

Reply:
There is no registration fee, and lunch and dinner are usually covered.

=>

For this first Hackathon, lunch and dinner will be covered and there will be no registration fee. These logistics are expected to evolve with the Hackathons. The NAMIC Project Week which has influenced the NWB Hackathon currently has a registration fee that participants pay, which covers breakfast, lunch, coffee and snacks during the day.